docs/classes/DocGeneratorFactory.md

```diff
@@ -1,379 +1,134 @@
-<embed> <a href="/docs/README.md">BumbleDocGen</a> <b>/</b> DocGeneratorFactory<hr> </embed>
-
-<h1>
-    <a href="https://github.com/bumble-tech/bumble-doc-gen/blob/master/src/DocGeneratorFactory.php#L18">DocGeneratorFactory</a> class:
-</h1>
-
+[BumbleDocGen](../README.md) **/**
+DocGeneratorFactory
 
+---
 
 
+# [DocGeneratorFactory](https://github.com/bumble-tech/bumble-doc-gen/blob/master/src/DocGeneratorFactory.php#L18) class:
 
 ```php
 namespace BumbleDocGen;
 
 final class DocGeneratorFactory
 ```
 
+## Initialization methods
 
+1. [__construct](#m-construct) 
+## Methods
 
+1. [create](#mcreate) - Creates a documentation generator instance using configuration files
```

Comment:
Same list issue here with everything having `1`.

Reply:
This is not an mistake, but the correct syntax for a numbered list :)